app/main.ipynb

```diff
@@ -563,25 +821,51 @@
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T14:11:19.696302Z",
-     "start_time": "2023-07-03T14:11:19.678876Z"
+     "end_time": "2024-11-29T23:26:42.368984Z",
+     "start_time": "2024-11-29T23:26:42.349221Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[0, 2, 0, 4, 0]"
+      ]
+     },
+     "execution_count": 81,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 81
   },
   {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-11-29T23:27:09.589265Z",
+     "start_time": "2024-11-29T23:27:09.576050Z"
+    }
+   },
    "cell_type": "code",
-   "execution_count": 30,
-   "outputs": [],
    "source": [
-    "# write your numpy code here"
+    "# write your numpy code here\n",
+    "np_lst = np.array(lst, dtype=np.int8)\n",
+    "np_mask = np.array(mask, dtype=bool)\n",
+    "np.where(np_mask, 0, np_lst)\n"
```

Comment:
The `np.where` function does not modify the original array `np_lst`. To achieve the intended effect, assign the result back to `np_lst` or use in-place modification.